src/userdel.c

```diff
@@ -1152,7 +1153,7 @@ int main (int argc, char **argv)
 	update_groups ();
 
 	if (rflg) {
-		errors += remove_mailbox ();
+		errors = remove_mailbox () && errors;
```

Comment:
This should be `||`, right?

Reply:
Oh, right. Thanks for the review! It's supposed to preserve true, not false.